src/Patient/Information/index.js

```diff
@@ -1,209 +1,69 @@
-    )
-}
-
-const TreatmentMessages = () => {
-    const classes = useStyles();
-    const { t } = useTranslation('translation');
-    const messages = t('treatmentUpdates', { returnObjects: true })
-
+export default function InformationPage() {
     return (
-        <div className={classes.treatmentMessages}>
-            {Object.keys(messages).map(each => {
-                return (
-                    <div key={each}>
-                        <span>{t('time.day')} {each}</span>
-                        <p>{messages[each]}</p>
-                    </div>)
-            })}
-        </div>
+        <Switch>
+            <Route path="/information/help">
+                <h1>Help</h1>
+            </Route>
+            <Route path="/information/videos">
+                <h1>Videos</h1>
+            </Route>
+            <Route path="/information/videos">
+                <h1>Help</h1>
+            </Route>
+            <Route path="/information/videos">
+                <h1>Report an Issue</h1>
+            </Route>
+            <Route>
+                <Buttons />
+            </Route>
+        </Switch>
     )
 }
 
-//Convert markdown file to expandable cards format
-const useStyles = makeStyles({
-    survey: {
-        padding: "1em",
-        "& > p": {
-            marginBottom: "1em"
-        }
-    },
-    container: {
-        "& > h1": {
-            marginBottom: "1em",
-            marginTop: "1em",
-            paddingLeft: "1em"
-        },
-        boxSizing: "border-box"
-    },
-    appInfo: {
-        paddingBottom: ".5em",
-        marginBottom: "1em",
-        paddingLeft: "1em",
-        display: "flex",
-        flexDirection: "column",
-        alignItems: "flex-start"
-
-    },
-    button: {
+const buttonData = [
+    { translationKey: "Videos", to: "/information/videos" },
+    { translationKey: "Help", to: "/information/help" },
+    { translationKey: "Questions and Answers", to: "/information/faq" },
+    { translationKey: "Report an Issue", to: "/information/issue" },
 
-        color: Colors.buttonBlue,
-        textTransform: "capitalize",
-        "& > span >  svg": {
-            marginRight: ".5em"
-        }
-    },
-    padding: {
-        paddingLeft: "1em"
-    },
-    treatmentMessages: {
-        display: "flex",
-        flexDirection: "column",
-        "& > div > span": {
-            textTransform: "capitalize",
-            color: Colors.textGray
-        }
-    },
-    help: {
-        display: "flex",
-        flexDirection: "column",
-        alignItems: "flex-start",
-        "& > h2": {
-            fontSize: "1em"
-        },
-        "& > button": {
-            marginLeft: "2em"
-        },
-        "& > button:first-of-type": {
-            marginLeft: 0
-        }
-    }
-})
+]
 
 
-export default Info;
+const Buttons = () => {
+    const classes = useStyles();
+    return (
+        <Box padding="16px">
+            <div className={classes.grid}>
+                {buttonData.map(each => {
```

Comment:
Nice. I like how you separated the buttonData and then mapped over it here. Is 'each' a normal  naming convention? jw. -jp